tests/toplevel_unit.c

```diff
@@ -57,6 +57,44 @@ DEFINE_TEST(range_contains) {
     roaring_bitmap_free(bm);
 }
 
+DEFINE_TEST(contains_multi) {
+    // create rbm with RLE conatainer form 0 1000
+    roaring_bitmap_t *bm = roaring_bitmap_from_range(0, 1001, 1);
+    // add array container from 66000
+    for (uint32_t i = 77000; i < 87000; i+=2) {
+        roaring_bitmap_add(bm, i);
+    }
+    // add bitset container from 20000
+    for (uint32_t i = 132000; i < 140000; i+=2) {
+        roaring_bitmap_add(bm, i);
+    }
+
+    bool results[10];
+    bool expected_results[10];
+    uint32_t values[10];
+    for (size_t i = 0; i < 10; i+=2) {
+        expected_results[i] = 1;
+        expected_results[i+1] = 0;
+    }
```

Comment:
You could make this less ad-hoc by simply storing the result of the regular `contains` function for `values[i]`.